<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugDialog</class>
 <widget class="QDialog" name="DebugDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">调试</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>551</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>URL</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="urlEdit">
        <property name="text">
         <string>https://www.kuaishou.com/short-video/3xmnuf55gxez5ca</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loadBtn">
        <property name="text">
         <string>加载</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="runJSBtn">
        <property name="text">
         <string>执行JS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="captureBtn">
        <property name="text">
         <string>截图</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="enableBtn">
        <property name="text">
         <string>设置可用</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>   JS代码</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>执行结果</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTextEdit" name="jsCodeEdit"/>
        </item>
        <item>
         <widget class="QTextEdit" name="jsResultEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
